main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -101,6 +111,31 @@ object Resolver extends Phase[NamedAst.Root, ResolvedAst.Root] {
 
   }
 
+  /**
+    * Performs name resolution on the given typeclass `c0` in the given namespace `ns0`.
+    */
+  def resolve(c0: NamedAst.Class, ns0: Name.NName, prog0: NamedAst.Root)(implicit flix: Flix): Validation[ResolvedAst.Class, ResolutionError] = c0 match {
```

Comment:
prog0 -> root

Reply:
I know you probably just copied the name from somewhere. This class is a bit "outdated" (but not broken!)